base/libgit2/remote.jl

```diff
@@ -88,3 +88,9 @@ function push{T<:AbstractString}(rmt::GitRemote, refspecs::Vector{T};
         !no_refs && finalize(sa)
     end
 end
+
+function name(rmt::GitRemote)
+    str_ptr = ccall((:git_remote_name, :libgit2), Cstring, (Ptr{Void}, ), rmt.ptr)
+    str_ptr == C_NULL && return nothing
```

Comment:
Also, `rmt` may be anonymous, in which case it has no `name`, right? This seems like a prime candidate for judicious application of `Nullable`...